californium-core/src/test/java/org/eclipse/californium/core/test/LinkFormatTest.java

```diff
@@ -0,0 +1,334 @@
+package org.eclipse.californium.core.test;
```

Comment:
Please add the eclipse license header (simply copy from other file and update the year and the organization you work for.